iXBRLViewerPlugin/viewer/src/js/viewer.js

```diff
@@ -196,15 +197,59 @@ Viewer.prototype._findOrCreateWrapperNode = function(domNode) {
     /* Otherwise, insert a <span> as wrapper */
     if (node.length == 0) {
         node = this._wrapNode(domNode);
+        node = node.find("*").addBack().filter(function () {
```

Comment:
I'm admittedly quite new to jQuery, but a small comment would have helped:
```
/* Select the node and all absolutely-positioned descendants. */
```
Also, I wonder if we should rename this to `nodes` as you did in `_preProcessiXBRL`?

Reply:
Done in 544b675 and 9ee05af.